README.md

```diff
@@ -229,7 +229,7 @@ Note that as part of the onboarding process your repository name may be normaliz
 [hashbang-grammar]: https://github.com/bmeck/proposal-hashbang
 [tests-hashbang-grammar]: https://github.com/tc39/test262/pull/1983
 [hashbang-notes]: https://github.com/tc39/tc39-notes/blob/master/es9/2018-11/nov-28.md#hash-bang-grammar
-[richer-keys]: https://docs.google.com/presentation/d/1q3CGeXqskL1gHTATH_VE9Dhj0VGTIAOzJ1cR0dYqDBk/edit#slide=id.p
+[richer-keys]: https://github.com/bmeck/proposal-richer-keys 
```

Comment:
Since this is stage 1, the repo needs to be transferred to the tc39 org; want to do that now?

Reply:
sure

Reply:
Also should this repo have a README if it's going to be linked here?

Reply:
@zenflow it would just point to the 2 parts in the repo, I could state that people should look at them but it would not have any explainer text at the top level

Reply:
@zenparsing added a landing readme